File: Cours 1/Python/cesar.py
import unidecode
message=input("Saisissez un message a chiffrer : ")
decalage=input("Saisissez le decalage : ").upper()
message=unidecode.unidecode(message).replace(" ","").upper()
def chiffre(lettre,decal):
    position_lettre=ord(lettre)-65
    position_decal=ord(decal)-65
    position_finale=(position_lettre+position_decal)%26
    return chr(65+position_finale)
message_chiffre=""
for i in message:
    message_chiffre+=chiffre(i,decalage)
print(message_chiffre)

File: Cours 1/Javascript/cesar.js
function chiffre_message(message,decalage){
    message_final="";
    for(i of message){
        position_lettre=i.charCodeAt(0)-65;
        position_decalage=decalage.charCodeAt(0)-65;
        position_finale=(position_lettre+position_decalage)%26;
        message_final+=String.fromCharCode(position_finale+65);
    }
    return(message_final);
}